### DISCUSSION TITLE
Nature’s coolest weapons

### DISCUSSION THREAD

**Discussion by erlend-sh** [fishfolk/jumpy#580] [Ideas]
_Please share your favorite animal weapons!_

**Comment by erlend-sh** [comment]
I’ll start: Any kind of shrimp.

![](https://s3.amazonaws.com/theoatmeal-img/comics/mantis_shrimp/mantis_shrimp2_5.png)

https://theoatmeal.com/comics/mantis_shrimp
https://www.wired.com/2014/07/absurd-creature-of-the-week-pistol-shrimp/

**Comment by olefasting** [comment]
(fungal) mind control, of course:

https://www.youtube.com/watch?v=vijGdWn5-h8

Honorable mention:
The rabies virus which, basically, turns you into an antisocial, blood-thirsty vampire

**Comment by 64kramsystem** [comment]
Silly idea: the whale opens the mouth and pushes or pulls or causes the enemy to jitter within a radius or line, while being still able to shoot. Thought of this while watching underwater animals with a big mouth :)
Actually, whale do sift lots of water, so a pull would be in line with the character :smile:

**Comment by SirGiraffey** [comment]
Morays are cool as enviromental enemies, they could be sort of hidden and when a fish comes cloes they pop out of sand and bite them.
![obraz](https://user-images.githubusercontent.com/57463655/127233527-9cc54a36-ad8d-4e46-920c-373b34115827.png)
![obraz](https://user-images.githubusercontent.com/57463655/127233585-4dc5a380-c474-4cb2-ae45-79f915a3b162.png)

**Comment by 64kramsystem** [comment]
Another one, somewhat similar to the moray - electric fish or pufferfish. They could go around the screen in "small mode", then, randomly, they emit electric shocks/inflate. This adds an interesting element of unpredictability, since one can't know if it's safe to stay close to the fish or not.

**Comment by erlend-sh** [comment]
https://fishkeepingforever.com/15-best-tropical-fish-for-beginners/

**Comment by erlend-sh** [comment]
![104E7AA9-B8C0-44D0-942D-DBA393626E4A](https://user-images.githubusercontent.com/583842/162917891-9fb96d00-c3b5-48a9-8c6c-055375ac2521.jpeg)
